Assets/Scripts/Utilities/RenderSettingsManager.cs

```diff
@@ -3,29 +3,25 @@
 using System;
 
 [Serializable]
-public class LightSettings
+public class TimeRenderSettings
 {
-	public Color color;
-	public float intensity;
-}
+	public Color color = Color.white;
 
-[Serializable]
-public class FogSettings
-{
-	public Color color;
-	public float density;
+	public Color lightColor = Color.white;
+	public float lightIntensity = 0.5f;
+
+	public Color fogColor = Color.white;
+	public float fogDensity = 0.05f;
```

Comment:
I'm cool with what you have. I was just curious if these are actual usable default values. I'd rather have "totally usable defaults" or "these are insane" instead of "these defaults are kind of okay". All these seem to fall into the former category so you're good.